subs/optimal_737.py

```diff
@@ -86,6 +86,7 @@ def get737_optimal_subs():
             '\\rho_{fuel}': 817.*units('kg/m^3'),  # Kerosene [TASOPT]
             'f_{wingfuel}': 1,  #.35 may be a better answer
             'TipReduct': 1,
+            '\\tau_{max_w}': 0.1267,
```

Comment:
@pgkirsch I went to a common wing box model for the full system model (PR #97 ) and in the process discovered a bug on this branch where the wing Nlift was set to 1. I fixed that with the is commit (the wingbox model is now the exact same as in PR #97 ).